sale_end_partner/models/account_move.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    end_partner_id = fields.Many2one(
```

Comment:
Can we change the field name to `partner_end_customer_id`, following the convention of other fields that references res.partner. Please apply the same idea throughout this module.

```suggestion
    partner_end_customer_id = fields.Many2one(
```

Reply:
Done!